src/desktop.md

```diff
@@ -372,43 +414,61 @@ In addition to that executable, you need the following:
 Place the DLL files in a directory next to the executable
 and the other DLLs, and bundle them together in a zip file.
 
+(FIX: Do you need a signing certificate of some kind?)
```

Comment:
I _believe_ the answer is no for this one, but you shouldn't take my word for it :)

Reply:
I can try this out this week and see.